src/ui/components/forms/VPNTextArea.qml

```diff
@@ -38,13 +35,16 @@ Item {
 
         TextArea.flickable: TextArea {
             property var maxCharacterCount: 1000
-            property bool loseFocusOnOutsidePress: true
+            property bool loseFocusOnBlur: true
```

Comment:
We'll need to update this line too https://github.com/mozilla-mobile/mozilla-vpn-client/blob/242e86cfb0862be5f172214fca96f8227eb949b6/src/ui/main.qml#L103 This may make more sense as `forceBlurOnOutsidePress` - it's actually a hack to forcefully remove focus from the input and close the system keyboard on iOS when the user taps anywhere outside the input.